openmetadata-ui/src/main/resources/ui/src/components/ProfilerDashboard/component/DataQualityTab.tsx

```diff
@@ -192,12 +209,14 @@ const DataQualityTab: React.FC<DataQualityTabProps> = ({
               <Tooltip
                 placement="bottomLeft"
                 title={
-                  hasAccess ? t('label.delete') : NO_PERMISSION_FOR_ACTION
+                  testCaseDeletePermission
+                    ? t('label.delete')
```

Comment:
same

Reply:
Here also no action is mentioned in the button so tooltip would be helpful.